src/components/Card.js

```diff
@@ -109,3 +102,11 @@ const InstructionStyled = styled.div`
   font-weight: bold;
   margin-top: 10px;
 `
+const ImageBinStyled = styled.img`
+  position: absolute;
+  height: 26px;
+  width: 30px;
+  opacity: 0.4;
+  bottom: 9px;
```

Comment:
Suggestion: Try to use standardize paddings and margins (for example 15px to bottom and right, also to Card-component)